wiki/People/Technical_Support_Team/en.md

```diff
@@ -0,0 +1,29 @@
+# Technical Support Team
+
+*For a team that administers accounts, see: [Account support team](/wiki/People/Account_support_team)*
+
+The **Technical Support Team**, also known as **Support Team Redux**, are members of the osu! staff whom are focused at forum moderation on several subforums: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (excluding [Tournaments](https://osu.ppy.sh/community/forums/55) and [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4), and [Help](https://osu.ppy.sh/community/forums/5).
+
+## Roles and responsibilities
+
+The Technical Support Team is responsible for:
+
+1. Testing and/or confirming bug reports to let the developers know and for them to resolve.
+2. Organising threads at [Feature Requests](https://osu.ppy.sh/community/forums/4) and [Help](https://osu.ppy.sh/community/forums/5), marking them as *Resolved*, *Confirmed*, *Invalid*, *Duplicate* and/or *Added*.
+
+If someone needs any assistance on the said subforums, members of this team are available to help.
+
+## Team members
+
+The [Technical Support Team group page](https://osu.ppy.sh/groups/22) lists all of the team members.
+
+*Note: All Technical Support Team members speak English in addition to the language(s) listed below unless noted otherwise.*
```

Comment:
this note should be right below the heading, so these two paragraphs should be swapped